packages/front-end/components/Modal.tsx

```diff
@@ -26,6 +26,7 @@ type ModalProps = {
   submit?: () => Promise<void>;
   secondaryCTA?: ReactElement;
   successMessage?: string;
+  showCTAs?: boolean;
```

Comment:
I don't like having this prop.  I think there's a better way to accomplish what you are trying to do